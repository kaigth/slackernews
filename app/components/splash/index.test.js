import React from 'react';
import Splash from './index';
import renderer from 'react-test-renderer';

it('renders the Loader correctly', () => {
  const tree = renderer
    .create(<Splash />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});