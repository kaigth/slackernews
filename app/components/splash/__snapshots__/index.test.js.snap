// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Splash overlay correctly 1`] = `
<div
  className="splash -hidden "
>
  <div
    className="_loader"
  />
</div>
`;
